README.md

```diff
@@ -6,7 +6,7 @@ Gringotts
 
 Gringotts is an item-based economy plugin for the Bukkit Minecraft server platform. Unlike earlier economy plugins, all currency value and money transactions are based on actual items in Minecraft, per default emeralds. The goals are to add a greater level of immersion, a generally more Minecraft-like feeling, and in the case of a PvP environment, making the currency itself vulnerable to raiding.
 
-[Get Gringotts from BukkitDev](http://dev.bukkit.org/server-mods/gringotts/)
+[Get Gringotts from Spigot](https://www.spigotmc.org/resources/gringotts.42071/)
```

Comment:
BukkitDev is still the "official" distribution. But we could link to both sites from the readme.

Reply:
That link is now a 404 so it should be changed anyway

Reply:
Yes, the new link is https://dev.bukkit.org/projects/gringotts

Reply:
Done 👍